### DISCUSSION TITLE
Click behaviour change for entry types viewed as cards.

### DISCUSSION THREAD

**Discussion by iainhenderson** [craftcms/cms#16185] [Ideas] [answered]
Editing related information in editor slide outs has been a fantastic improvement. It occurred to me recently that many authors may not discover this powerful feature as it is hidden behind non intuitive actions.

Currently an author edits an entry type in a slide out UI by double clicking the background of a card. (or through the ellipsis dropdown) While clicking on the bold blue title opens the entry as a page - losing the current context.

- Double clicking is a uncommon action on the web.
- There is no suggestion clicking on the card does anything (no hover behaviour or cursor style)
- The blue title is very prominent, and users tend to click on text as links, making this the primary click target.

I believe authors would generally find it more useful to edit entry types in the slide out UI, and that this should be the primary behaviour. Clicking the title should open the slide out UI, and there should be a less prominent way to open the entry as a page. I suggest adding the break-out button to card displays, which is already in use in other areas of the UI.

![Break out](https://github.com/user-attachments/assets/f0a6634f-852c-45a0-8860-0b3079e9ce09)

When viewed as inline cards, a drop arrow could replace the break out to trigger the accordion expand/collapse, which is also another piece of useful functionality hidden by a double click.

![Collapsable inline card](https://github.com/user-attachments/assets/9f059a1e-4335-4a86-af7d-0124c4df2983)

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the feedback! Addressing this in Craft 5.7 via #16983.

**Reply by brandonkelly** [reply]
Craft 5.7.0 is out with that change.
